common/compression/clickteam.cpp

```diff
@@ -213,7 +434,34 @@ ClickteamInstaller* ClickteamInstaller::open(Common::SeekableReadStream *stream,
 	if (block3_offset <= 0 || block3_len <= 0)
 		return nullptr;
 
-	return new ClickteamInstaller(files, tags, crc_xor, block3_offset, block3_len, stream, dispose);
+	if (verifyOriginal && reference) {
+		for (Common::HashMap<Common::String, ClickteamFileDescriptor, Common::IgnoreCase_Hash, Common::IgnoreCase_EqualTo>::iterator i = files.begin(), end = files.end();
```

Comment:
auto?

Reply:
Style guide and similar places in other archives are against auto. I'd prefer to  be consistent